src/components/WorkspaceList/WorkspaceListConfig.tsx

```diff
@@ -60,15 +60,19 @@ export const workspaceFilters = [
 ];
 
 // Actions on each row
-export const workspaceActions: IAction[] = [
+export const workspaceActions = ({ deleteWorkspace }: { deleteWorkspace: (workspaceName: string) => void }): IAction[] => [
   {
     title: 'Edit workspace',
     onClick: () => null,
   },
   {
     title: 'Delete workspace',
-    onClick: () => null,
+    onClick: (event, rowIndex, rowData /*, extraData */) => {
+      //  console.log('Event', event, ' | rowIndex', rowIndex, ' | rowData', rowData, ' | extraData', extraData);
+      // deleteWorkspace('kk');
+      deleteWorkspace(rowData.name);
```

Comment:
Right now this isn't working because rowData is not passed back.  If you want to test this, hard code the name of workspace to be deleted (see comment on line ~72)